core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java

```diff
@@ -329,7 +329,7 @@ String generateSep10Jwt(String challengeXdr, String clientDomain)
     Memo memo = challenge.getTransaction().getMemo();
     Sep10Jwt sep10Jwt =
         Sep10Jwt.of(
-            sep10Config.getWebAuthDomain(),
+            "https://" + sep10Config.getWebAuthDomain() + "/auth",
```

Comment:
Can the `web_auth_domain` prefixed with `https://`?

Reply:
Yeah SEP-10 expects url, not just domain.
This will always be a valid URL for production.